### DISCUSSION TITLE
Parameterization PEG esters

### DISCUSSION THREAD

**Discussion by shakhawath** [marrink-lab/polyply_1.0#336] [Martini3]
Hello,
Are there any tutorials I should follow to create topology and structure files for PEG monoesters and PEG diesters? 
Similar to Pegylated lipids, I want to medium chain fatty acids with PEG08. I would be glad to get some suggestions.
Thanks,
Shakhawath.

**Comment by fgrunewald** [comment]
@shakhawath if I understand you correctly you want to PEGylate a fatty acid, which makes the acid to an ester? Do you already have run an all-atom simulation? Can you perhaps share the chemical structure?

**Comment by shakhawath** [comment]
Hello, Thank you for your quick reply.
No, I have not run the AAMD simulations. I am attaching here the structures below:
![PEF_esters](https://github.com/marrink-lab/polyply_1.0/assets/31314367/9d43b465-5e84-4b2b-a62a-f898aa4ba92b)
Thanks.

**Comment by fgrunewald** [comment]
@shakhawath lucky for you I was able to collect parameters from other projects and add them to a polyply input file such that you get parameters for your molecules with at least a very good guess. If you want you can refine them but I suggest to test them first and if there are no major issues no refinement is needed.

In order to generate the parameters you have to download [EST.ff.txt](https://github.com/marrink-lab/polyply_1.0/files/12266990/EST.ff.txt) this file and rename it to EST.ff. Subsequently, use the following command with polyply to generate the itp file for the ester and dieter respectively:

Monoester:
```
polyply gen_params -f EST.ff -lib martini3 -o mono_est.itp -seq PE:4 EST:1 PEO:7 OHter:1 -name PEG_EST
``` 
Diester:
```
polyply gen_params -f EST.ff -lib martini3 -o diester.itp -seq PE:4 EST:1 PEO:6 EST:1 PE:4 -name PEG_EST
```
Please note that in this case represent 4 carbon atoms of the tail as one PE unit. So you need to count the number of carbons in R and divide that by 4 to get the number of PE units. In the example case, the tail would have 16 carbon atoms. Also, note that PEG here is given as a COC repeat unit and that the OH group is represented explicitly. Therefore, in the first case, you have 7 PEGs plus 1 OH end-group. In the second case, you have 6 because 1 unit in your drawing is counted within the ester. 

Let me know if the parameters work out! In case you want to refine them let me know and I point you towards the relevant tutorials.

**Comment by shakhawath** [comment]
@fgrunewald Thank you very much for the help. I could make the structures, and run some sample test simulations and it seems like the parameters are working fine. I will think about validating the models soon. 
Thanks.

**Comment by fgrunewald** [comment]
@shakhawath happy to hear that the model and coordinate generation worked. Please let me know if the validations work out as well. For the time being, I'm moving this issue to the [discussions section](https://github.com/marrink-lab/polyply_1.0/discussions). This way it will remain accessible for further exchange about the PEG ester models. Additionally, please don't forget to give us a star if you're happy with the package and are using it in your research. It really helps us to justify the work we put into maintaining the package.

**Comment by Lynn3512138** [comment]
@fgrunewald Dear Dr. Grunewald:
This discussion is super helpful, and I’m working on building an ester-linked PEG chain. In my case, however, the alkyl (PE unit) part of the ester cannot be divided by 4 directly, so I plan to use a smaller bead representing a 3-carbon unit. It seems to me that this approach will require reparameterization of the bond parameters for the linkage. My plan is to build an AA diester-PEG chain in either water or vacuum (I’m unsure which is better in this case) and run the simulation for a sufficiently long time to allow the chain to explore its configurations. Then, following the MARTINI tutorial, I’ll generate new bond parameters based on the sampled data.
Do you have any suggestions on:
1. Whether to simulate in water or vacuum? 2. How long the simulation should run to ensure adequate sampling? 3.Any other considerations for reparameterization in this context?
Thank you very much!
